phpBB/language/en/acp/posting.php

```diff
@@ -54,6 +54,10 @@
 	'BBCODE_HELPLINE_TEXT'		=> 'Help line text',
 	'BBCODE_HELPLINE_TOO_LONG'	=> 'The help line you entered is too long.',
 
+	'BBCODE_ICON_NAME'			=> 'Button icon',
+	'BBCODE_ICON_NAME_EXPLAIN'	=> 'Enter the name of a Font Awesome icon to display instead of the BBCode name appearing on the button. Do not include the fa prefix.',
```

Comment:
Maybe add a link to the icon list here

Reply:
phpBB 3.3.x uses FA [4.7.0](https://fontawesome.com/v4/icons/)

Reply:
Maybe this? 🤔 

https://fontawesome.com/v5/search